content/docs/command-reference/root.md

```diff
@@ -29,22 +30,78 @@ refer to a file in another directory.
 
 ```dvc
 $ dvc root
-
 .
 
 $ cd subdir
 $ dvc root
-
 ..
 ```
 
 ## Example: Referencing files
 
+Simplify file references when working in a subdirectory of a <abbr>DVC
+project</abbr>.
```

Comment:
Nice use of abbr 👍 

Maybe add something like "...by making all paths relative to the root dir?" to explain what you mean by "simplify"?

Let's be explicit 🙂

Reply:
Updated this and made consistent between examples.

Reply:
Looks good, thanks.